app/models/organisation.rb

```diff
@@ -105,11 +105,12 @@ def display_provider_type
   def display_organisation_attributes
     attrs = [
       { name: "Name", value: name, editable: true },
+      { name: "Organisation ID", value: "ORG#{id}", editable: false },
```

Comment:
I know this is more a point on the ticket itself, but is there any risk that users will copy and paste this value, attach it to support tickets, try searching it etc and run into issues as it's just amended on a surface level? 
- Can the actual ID be a string and have it amended at an earlier point
- Is this documented clearly somewhere
- Can ORG be appended to the end of the field title so that when highlighted only the number is selected
- Or is this not potentially an issue at all :)

Reply:
So the id here is prepended with ORG because there are actually two types of org IDs - orgs from old CORE have a different numbering, so every org in new CORE is prepended with ORG so that's clear in bulk upload, as users can do bulk upload for old core OR new core org ids depending on if they just give a number or ORG then a number.
We shouldn't amend ID much earlier than here (maybe there's an argument for a helper but it's not even conditional so I think this is fine in the model) because we went to be passing IDs as ints where possible throughout the codebase for convenience.
I agree with your point about users searching for orgs - except I don't think there's anywhere in the service where they can actually do that, you can't search for orgs in logs and you search by org name as a support user, so I think that's probably fine too.

Reply:
re the documentation question, I would not at all be surprised if this is not documented clearly anywhere, I was mainly referring to the codebase to check this is consistent with the bulk upload pattern - I'll see if I can find it in docs anywhere/add it if not, good idea!